src/settings/Settings.js

```diff
@@ -213,11 +213,15 @@ export const SETTINGS = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
         default: "en",
     },
-    "analyticsOptOut": {
+    "analyticsOptIn": {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG,
```

Comment:
Something to consider may be to prevent the configuration from setting this flag. `WITH_CONFIG` means the `config.json` is able to set a default value that overrides this.

There are arguments for both keeping and ditching the config-level option, just thought I'd point it out.

Reply:
This is as intended, allowing config.json to override both seems like a legitimate use case.